### PR TITLE
PLANET-7365: Clean up Listing page code

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-7365

# Description
Remove duplicated, unused and obsolete code related to listing pages

Clean up NROs code to make that nothing will be broken after we merge the current PR
- https://github.com/greenpeace/planet4-child-theme-hongkong/pull/32
- https://github.com/greenpeace/planet4-child-theme-korea/pull/21
- https://github.com/greenpeace/planet4-child-theme-taiwan/pull/30
- https://github.com/greenpeace/gpea-planet8-child-theme/pull/83

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [oberon](https://www-dev.greenpeace.org/test-oberon) | [admin](https://www-dev.greenpeace.org/test-oberon/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-oberon/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-oberon) | [composer-local.json](https://github.com/greenpeace/planet4-test-oberon/blob/main/composer-local.json)

:watch: 2024.06.14 19:05:10

File: author.php [hunk 1]
```diff
@@ -34,14 +34,7 @@
         'width' => '300',
         'height' => '300',
     ];
-
```
Comment (GP-Dan-Tovbein) [inline author.php:37]: Noticing that this value is not being used by us, only by a couple of NROs. Do we really need to add it? What if we ask them to include into their themes instead?
https://github.com/search?q=org%3Agreenpeace%20author_share_buttons&type=code

Comment (mleray) [reply author.php:37]: I think @comzeradd would know better about this, especially if we need to warn the NROs about the change!

**Review by mleray — APPROVED** [review]
Tested on the instance, looks good to me 👍

**Pull request merged**